app/views/people/edit.html.haml

```diff
@@ -10,12 +10,27 @@
         %ul
           - @person.errors.full_messages.each do |msg|
             %li= msg
-.extra-spacing.row
+
+.row
   .col.s12
-    %h5 Accountgegevens bewerken
-.extra-spacing.row
-  .col.s12
-    = form_for(@person) do |f|
+    = form_for(@person, html: { class: 'settings' }) do |f|
+      %h5 Bankgegevens
+      %p{style: @person.iban.blank? ? 'color:red' : ''}
+        Om de beloning over te maken hebben we nog je IBAN gegevens nodig.
+      %p
+        Dit wordt in een aparte database opgeslagen, zodat de vragen die je net hebt beantwoord alsnog helemaal anoniem zijn.
+      %p
+        Let op: wij verwijderen al je persoonlijke gegevens, dus je naam, telefoonnummer en IBAN, op 31 augustus 2018. Hierna is het niet meer mogelijk om correcties te doen in het uitbetalen, want dan hebben we je gegevens niet meer. Trek daarom op tijd aan de bel als je denkt dat er iets mis is gegaan (mail
+        %span>= link_to " #{ENV['INFO_EMAIL']}", "mailto:#{ENV['INFO_EMAIL']}"
+        ), je hoort het bedrag voor 17 augustus te ontvangen.
+      %br
+      .input-field
+        = f.label :iban
```

Comment:
moeten we dan niet ook voor / achternaam hierbij zetten? Althans, die moeten ze eigenlijk ook checken lijkt me?

Reply:
Fixed.